pkgs/dartpad_ui/lib/main.dart

```diff
@@ -682,7 +685,7 @@ class DartPadAppBar extends StatelessWidget implements PreferredSizeWidget {
     url_launcher.launchUrl(Uri.parse(response.idxUrl));
   }
 
-  Future<void> _generateNewCode(BuildContext context) async {
+  Future<void> _generateNewCode(BuildContext context, AppType? appType) async {
```

Comment:
can we make `appType` non-nullable? i.e., do we ever need to pass in null here

Reply:
if so (if we do need to preserve null) then lets make it an optional named param

Reply:
Yes, it will become non-nullable in the following PRs